tools/ClassGen/InstrBuilder.cpp

```diff
@@ -214,6 +214,21 @@ void InstrBuilder::emitClass(std::ostream &os) const {
     os << "  void verify() const;\n";
   } else {
     os << "  void verify() const {\n";
+
+    // All non-memory instructions should have the property that all operands
```

Comment:
What do you mean by non-memory instructions? Do quantization-profile instructions fall into this category?